test-cases/longevity/longevity-twcs-48h.yaml

```diff
@@ -25,7 +25,7 @@ space_node_threshold: 64424
 
 user_prefix: 'longevity-twcs-48h'
 
-post_prepare_cql_cmds: "ALTER TABLE scylla_bench.test with gc_grace_seconds = 300 and default_time_to_live = 21600 and compaction = {'class':'TimeWindowCompactionStrategy', 'compaction_window_size': 60, 'compaction_window_unit': 'MINUTES'};"
+post_prepare_cql_cmds: "ALTER TABLE scylla_bench.test with gc_grace_seconds = 15000 and default_time_to_live = 21600 and compaction = {'class':'TimeWindowCompactionStrategy', 'compaction_window_size': 60, 'compaction_window_unit': 'MINUTES'};"
```

Comment:
what are you expecting by increasing the GC grace from 300 to 15K ?
what is the issue you are trying to solve with it?

Reply:
the tll is 6 hours, the window size is  1 hour, it means that we will have about  6 sstables per table. increasing gc to ~4h i want to avoid frequency of checks for tombstone clearing. It should help to avoid sb read errors

Reply:
that information will be very useful in the commit message

Reply:
> @roydahan please note @juliayakovlev marked this fix related to [scylladb/scylla-bench#95](https://github.com/scylladb/scylla-bench/issues/95) but i'm not sure it is related to the original scenario of it that doesn't include TTL.

@yarongilor 
I mentioned this fix in the issue because of Alex suppose that this is the problem in the test. But it's not